api/fees/data.go

```diff
@@ -148,22 +148,27 @@ func (fd *FeesData) getRewardsForCache(block *block.Block) (*rewards, error) {
 	}
 	state := fd.stater.NewState(parentSummary.Root())
 
-	baseGasPrice, err := builtin.Params.Native(state).Get(thor.KeyBaseGasPrice)
+	legacyTxDefaultGasPrice, err := builtin.Params.Native(state).Get(thor.KeyLegacyTxDefaultGasPrice)
 	if err != nil {
 		return nil, err
 	}
 
 	// Get the effective priority fee (reward) for each transaction
 	transactions := block.Transactions()
 	items := make([]rewardItem, len(transactions))
-	isGalactica := header.Number() >= thor.GetForkConfig(fd.repo.NewBestChain().GenesisID()).GALACTICA
+
 	for i, tx := range transactions {
 		provedWork, err := tx.ProvedWork(header.Number(), fd.repo.NewBestChain().GetBlockID)
 		if err != nil {
 			return nil, err
 		}
 		items[i] = rewardItem{
-			reward:  fork.GalacticaPriorityPrice(tx, baseGasPrice, provedWork, &fork.GalacticaItems{IsActive: isGalactica, BaseFee: header.BaseFee()}),
+			reward: fork.GalacticaPriorityGasPrice(
```

Comment:
@freemanzMrojo An off-topic question, can `reward` here incorporate `receipt.reward/receipt.gasUsed`? The `provedWork` above will require computation resource .

Reply:
Good point - Pushed it to offline convo.